src/overlays/actors/ovl_En_Floormas/z_en_floormas.h

```diff
@@ -4,10 +4,29 @@
 #include <ultra64.h>
 #include <global.h>
 
-typedef struct {
+#define SPAWN_INVISIBLE 0x8000
+#define SPAWN_SMALL 0x10
+
+// Merge params
+#define MERGE_MASTER 0x40
+#define MERGE_SLAVE 0x20
```

Comment:
If you're exposing these in the header for external use, then they should probably have a unique name that includes "FLOORMAS" or something.